src/routes/api/index.js

```diff
@@ -3,6 +3,6 @@ import users from './users';
 
```

Comment:
Why did you delete the `models/users.js` file?

Reply:
I renamed only renamed it to avoid  sequelize errror

Reply:
how does it cause sequelize error?

Reply:
The order of arrangement in the migrations folder. Sequelize was throwing error because it was trying to create role roles which has a foreign key from users

Reply:
if they were properly done you shouldn't have that issue.